Multisig/Features/Security/SecurityCenter.swift

```diff
@@ -74,66 +74,106 @@ class SecurityCenter {
     }
 
     // import data potentially overriding existing value
-    func `import`(id: DataID, ethPrivateKey: EthPrivateKey, completion: @escaping (Result<Bool?, Error>) -> ()) {
-        perfomSecuredAccess { [unowned self] result in
-            switch result {
-            case .success:
-                do {
-                    try sensitiveStore.import(id: id, ethPrivateKey: ethPrivateKey)
-                    completion(.success(true))
-                } catch let error {
-                    completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
+    func `import`(id: DataID, ethPrivateKey: EthPrivateKey, protectionClass: ProtectionClass = .sensitive, completion: @escaping (Result<Bool?, Error>) -> ()) {
+        switch(protectionClass) {
+        case .sensitive:
+            perfomSecuredAccess { [unowned self] result in
+                switch result {
+                case .success:
+                    do {
+                        try sensitiveStore.import(id: id, ethPrivateKey: ethPrivateKey)
+                        completion(.success(true))
+                    } catch let error {
+                        completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
+                    }
+                case .failure(let error):
+                    completion(.failure(error))
                 }
-            case .failure(let error):
-                completion(.failure(error))
+            }
+        case .data:
+            do {
+                try dataStore.import(id: id, ethPrivateKey: ethPrivateKey)
+                completion(.success(true))
+            } catch let error {
+                completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
             }
         }
     }
 
-    func remove(address: Address, completion: @escaping (Result<Bool?, Error>) -> ()) {
-        perfomSecuredAccess { [unowned self] result in
-            switch result {
-            case .success:
-                do {
-                    try sensitiveStore.delete(address: address)
-                    completion(.success(true))
-                } catch let error {
-                    completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
+    func remove(address: Address, protectionClass: ProtectionClass = .sensitive, completion: @escaping (Result<Bool?, Error>) -> ()) {
```

Comment:
unrelated comment: 

```(Result<Bool?, Error>) -> ()``` means it returns `Result`. Result is an enum with 2 cases: `success` and `failure`. In this declaration, for success case, it returns boolean that can be nil. I understand when the `success(true)` means success, but what do `success(false)` or `success(nil)` mean?

If `success` case's Bool is always true, then it can be replaced with `Void` type (which is equivalent to empty tuple `()`). Then, to indicate success, you'd do `success(())`. I do agree that is less readable.

Another alternative is to return `Error?` and if it's nil, that would mean success.